setup.py

```diff
@@ -13,24 +15,62 @@
 with open(os.path.join(BASE_DIR, "deflate.c"), "r") as src:
     version = re.match(r'.*#define MODULE_VERSION "(.*?)"', src.read(), re.S).group(1)
 
-cpu = "arm" if platform.machine().startswith("arm") else "x86"
+libdeflate_found = False
+
+try:
+    from pkgconfig import installed as pkgconfig_installed
+    from pkgconfig import parse as pkgconfig_parse
+except ImportError:
+    # pkgconfig is not installed. It will be installed by setup_requires.
+    pass
+else:
+    def pkgconfig_installed_check(lib, required_version, default):
+        installed = default
+        try:
+            installed = pkgconfig_installed(lib, required_version)
+        except EnvironmentError:
+            # no pkg-config present
+            pass
+        except ValueError:
+            # pkgconfig was unable to determine if required version of
+            # libdeflate is available. Bundled version of libdeflate will be
+            # used.
+            pass
```

Comment:
I guess one would want to emit some warning, so the user will know what's missing / how to fix that.

Reply:
it also could emit a info msg when it has found libdeflate via pkg-config and will use that.

Reply:
hmm, if you always fall back to bundled code if pkg-config did not work, it's maybe rather a info msg in all cases.
just tell the user whether it used the pkg-config located system lib or the bundled stuff because ...